MapboxNavigation/Constants.swift

```diff
@@ -111,3 +111,9 @@ public func RouteControllerMinimumDistanceForHighAlert(identifier: MBDirectionsP
  When calculating whether or not the user is on the route, we look where the user will be given their speed and this variable.
 */
 public var RouteControllerDeadReckoningTimeInterval:TimeInterval = 1.0
+
+
+/*
+ Maximum angle the user puck will be rotated when snapping the user's course to the route line.
+*/
+public var RouteControllerMaxManipulatedCourseAngle:CLLocationDirection = 25
```

Comment:
CLLocationDirection is used for directions, not direction _deltas_. A CGFloat or Double would’ve sufficed here, along with replacing “angle” with “degreeOffset” and replacing “max” with “maximum”.